kadi/commands/observations.py

```diff
@@ -662,54 +741,34 @@ def get_agasc_cone_fast(ra, dec, radius=1.5, date=None, matlab_pm_bug=False):
         Date for proper motion (default=Now)
     matlab_pm_bug : bool
         Apply MATLAB proper motion bug prior to the MAY2118A loads (default=False)
+    agasc_file : str, None
+        AGASC file name (default=None)
 
     Returns
     -------
     Table
-        Table of AGASC entries
+        Table of AGASC entries with AGASC_ID, RA, DEC, PM_RA, PM_DEC, EPOCH, MAG_ACA,
+        RA_PMCORR, DEC_PMCORR columns.
     """
-    global STARS_AGASC
```

Comment:
Looks like this is still initialized at the top of the file?  Not sure how that survived ruff.

Reply:
Ruff allows initializing a constant that is not used in the code since it could be part of the API, or whatever. Anyway I fixed this.